main_test.go

```diff
@@ -51,7 +51,7 @@ func TestBinarySize(t *testing.T) {
 	// so.
 	//
 	// When increasing, use current binary size on amd64 + 1M.
-	const maxSize int64 = 9103464
+	const maxSize int64 = 11357152
```

Comment:
Doesn't it link dynamically to OpenSSL? Is this size increase really just from the glue code between the two?

Reply:
You are correct. It links dynamically. At least the version i have on my armhf target does that.
This size increase worries me.

Reply:
@kacf i have removed the dependency to spacelog from openssl now, and that allows
us to drop this patch. Things look sane now.